exonum-java-binding/core/src/test/java/com/exonum/binding/core/storage/indices/ProofListContainsMatcher.java

```diff
@@ -18,15 +18,18 @@
 
 import static com.google.common.base.Preconditions.checkArgument;
 import static com.google.common.base.Preconditions.checkNotNull;
+import static java.util.stream.Collectors.toList;
+import static java.util.stream.Collectors.toMap;
 import static org.hamcrest.core.IsEqual.equalTo;
 
+import com.exonum.binding.common.collect.MapEntry;
 import com.exonum.binding.common.proofs.list.CheckedListProof;
 import com.exonum.binding.common.proofs.list.UncheckedListProof;
-import java.util.Collections;
-import java.util.List;
-import java.util.Map;
-import java.util.TreeMap;
+
+import java.util.*;
```

Comment:
😱

Reply:
Sorry, my IDEA annoyingly does that, couldn't get myself to reconfigure that. Just did though! Thanks.